sections/04_study.md

```diff
@@ -264,121 +264,236 @@ and [#258](https://github.com/greenelab/deep-review/issues/258).`
 *We may want to be selective about what we discuss and not list every
 application in this area.*
 
+`TODO: There's inevitably a lot of overlap been this and the transcription
+binding section. Maybe in the long term think about restructuring to one big
+"regulome" section`
+
+`TODO: There's a mass of papers that could be referred to here, so some
+distilling or organizing scheme is perhaps needed.`
+
+Identification of promoters and other cis-regulatory elements (CREs) like
+enhancers presents an obvious use case for deep learning. Transcription control
+is undoubtedly a vital - and early - part of the regulation of gene expression.
+An abundance of upstream sequence and associated functional data (e.g. ENCODE,
+ExAC) exists across species. Yet studies of gene regulation have often focused
+on the protein (binding) rather than promoter level [@doi:10.1.1.108.1249],
+perhaps due to the ill-defined nature of CREs. A promoter itself can be seen as
+an assemblage of "active" binding sites for  transcription factors interspersed
+by less-characterized and perhaps functionally silent spacer regions. However,
+the sequence signals that control the start and stop of transcription and
+translation are still not well understood, compounded by incomplete
+understanding of alternative transcripts and the context for these alternatives.
+Homologs might be studied but they may not exist and their promoters might not
+be characterized either. Sequence similarity is poor even between functionally
+correlated genes.
+
+Recognizing enhancers presents additional challenges. Enhancers may be up to 1
+Mbp upstream or downstream from  the target promoter, on either strand, even
+within the introns of other genes [@doi:10.1038/nrg3458]. They do not
+necessarily operate on the nearest gene and may in fact effect multiple genes.
+Their activity is frequently tissue- or context-specific. A substantial fraction
+of enhancers displays modest or no conservation across species. There is no
+universal enhancer sequence signal or marker for enhancers, and some literature
+suggests that enhancers and promoters may be just categories along a spectrum
+[@doi:10.1016/j.tig.2015.05.007]. One study [@doi:10.1101/gr.173518.114.] even
+showed that only 33% of predicted regulatory regions could be validated, while a
+class of "weak" predicted enhancers were strong drivers of expression.
+classified as positive in the experiment Yet there is growing evidence for their
+vast ubiquity, making them possibly the predominant functional non-coding
+element. Thus, identifying enhancers is critical yet the the search space is
+embarrassingly large.
+
+While prior (non-deep learning) approaches have made steady improvements on
+promoter prediction [@doi:10.1101/gr.7.9.861], there is little consensus on the
```

Comment:
@dhimmel Any idea why this DOI can't be resolved?  It matches what I see here http://genome.cshlp.org/content/7/9/861.extract